src/DataGrid.tsx

```diff
@@ -722,6 +723,10 @@ function DataGrid<R, SR>({
   }
 
   function navigate(event: React.KeyboardEvent<HTMLDivElement>) {
+    if (selectedPosition.mode === 'EDIT') {
+      const onNavigation = columns[selectedPosition.idx].editorOptions?.onNavigation ?? onEditorNavigation;
```

Comment:
We can probably add an example of custom `onEditorNavigation` like 
- Navigate left if the cursor is at position 0
- Navigate right if the cursor is at position selection.length